frontend/src/global_styles/layout/work_packages/_details_view.sass

```diff
@@ -120,7 +120,7 @@ body.router--work-packages-partitioned-split-view-new
   display: flex
   flex-direction: column
   margin-right: 4px
-  max-width: 100%
+  overflow: hidden
```

Comment:
This prevents the dropdown of the type selector to be visible which is also the reason why the test is failing.
<img width="158" alt="Bildschirmfoto 2021-11-18 um 13 20 03" src="https://user-images.githubusercontent.com/7457313/142414446-4aba51aa-0884-4be8-a3ab-b884a3656df5.png">